### PR TITLE
Password Hide and show

### PR DESCRIPTION
**Pull request by VivianVKJ**

### CODE REVIEW

File: src/screens/LoginScreen/LoginScreen.tsx [hunk 1]
```diff
@@ -1,61 +1,78 @@
...
+            value={password}
+            secureTextEntry={hidePwd}
+            onChangeText={text => setPassword(text)}
+            style={styles.input}
+            underlineColor="gray"
+            autoCapitalize="none"
+          />
+        </View>
+        <View style={{ flex: 2, marginBottom: 15 }}>
+          {/* <Button title="Show" onPress={() => Alert.alert("Button")} /> */}
```
Comment (FoodIsLifeBGP) [inline src/screens/LoginScreen/LoginScreen.tsx]: Please take out commented code, unless it's needed for a text comment.

File: src/screens/LoginScreen/LoginScreen.tsx [hunk 2]
```diff
@@ -1,61 +1,78 @@
-import React, { useState } from 'react';
-import { View, Alert } from 'react-native';
-import { TextInput } from 'react-native-paper';
-import login from '../../util/login';
-import { Title, LinkButton, SpacerInline } from '../../elements';
-import InputLabel from '../../elements/FormTextInput/InputLabel';
-import styles from './LoginScreen.styles';
-import { useNavigation, useNavigationParam } from 'react-navigation-hooks';
+import React, { useState } from "react";
```
Comment (FoodIsLifeBGP) [inline src/screens/LoginScreen/LoginScreen.tsx]: Please leave in the single quotes here, double quotes (outside of React components) are a signal that the string contains a single quote.  I'm working on a linting setup that will make these guidelines more obvious in the future.

File: src/screens/LoginScreen/LoginScreen.tsx [hunk 3]
```diff
@@ -1,61 +1,78 @@
...
+import styles from "./LoginScreen.styles";
+import { useNavigation, useNavigationParam } from "react-navigation-hooks";
 
 export default () => {
-	const { navigate } = useNavigation();
-	const [ email, setEmail ] = useState(useNavigationParam('email') || '');
-	const [ password, setPassword ] = useState(useNavigationParam('password') || '')
+  const { navigate } = useNavigation();
+  const [email, setEmail] = useState(useNavigationParam("email") || "");
+  const [password, setPassword] = useState(
```
Comment (FoodIsLifeBGP) [inline src/screens/LoginScreen/LoginScreen.tsx]: Why the formatting change on this one but not the others?

Comment (VivianVKJ) [reply src/screens/LoginScreen/LoginScreen.tsx]: My fault, I used an extension on VS code which automatically change the quotes and this line. I've already changed them back

**Pull request merged**